### DISCUSSION TITLE
Functionality issue

### DISCUSSION THREAD

**Discussion by BizToolsCanada** [alexey-gamov/grandstream-ctc#1] [General]
Hi,
Seems the extension works only when I have a tab open and logged in to the telephone web interface. Is there a way around this?
Thanks.

**Comment by alexey-gamov** [comment]
Hi! Extension is not using opened phone web ui. 

It's makes POST and GET requests for original APIs.

Can give some info?

- Phone model and FW version.
- Dev output (F12) from extension, i think it's called "background page" in extension list.

**Comment by BizToolsCanada** [comment]
I understand it does not use the open phone webb ui. I found the extenssion did not work (dial or answer call) unless there is a tab with phone ui (logged in).

Product Model	GXP2170
Part Number	9620006914C
Individual Certificate Generation	
GEN 1 ( SHA 256 + RSA )
Software Version
Boot	1.0.7.2
Core	1.0.11.45
Base	1.0.11.47
Prog	1.0.11.71
Locale	1.0.11.61
Recovery	1.0.11.46

I'm not sure where to find the Dev output.

**Reply by alexey-gamov** [reply]
Check page 6 of [CTI manual](https://www.grandstream.com/hubfs/Product_Documentation/CTI_Guide.pdf), may be something is different on your phone.

Do you have telegram account?
May be pm me [@alexey_gamov](https://t.me/alexey_gamov)
